rl_algorithms/dqn/agent.py

```diff
@@ -135,6 +134,7 @@ def _init_network(self):
 
         self.dqn = BaseNetwork(self.backbone_cfg, self.head_cfg).to(device)
         self.dqn_target = BaseNetwork(self.backbone_cfg, self.head_cfg).to(device)
+        self._get_dqn_loss = build_loss(self.hyper_params.loss_type)
```

Comment:
should change the variable name: `self.loss_fn`

Reply:
resolved at 2113f91!